### ISSUE TITLE
Uncaught TypeError with redux-devtools

### ISSUE THREAD

**Issue by juangl** [chentsulin/electron-react-boilerplate#259]
I get the next error:

```
$ npm run start-hot

> electron-react-boilerplate@0.10.0 start-hot /home/jn/electron-react-boilerplate
> cross-env HOT=1 NODE_ENV=development electron -r babel-register -r babel-polyfill ./main.development

[15544:0628/225148:ERROR:CONSOLE(20584)] "Uncaught Error: Cannot find module "./App"", source: chrome-extension://redux-devtools/js/devpanel.bundle.js (20584)

```

**Comment by jhen0409** [comment]
It looks redux-devtools-extension v2.2.0 changes, the `chrome.storage.local` is not supported on Electron. I've send a PR for that.

**Comment by juangl** [comment]
Thanks, @jhen0409 for comment. I edited the log with redux-devtools-extension updated. The log is a little different.

**Comment by jhen0409** [comment]
It seems my fault, and @zalmoxisus has not checked. 😂

Please wait https://github.com/zalmoxisus/redux-devtools-extension/pull/151.

**Comment by jhen0409** [comment]
It should fixed in redux-devtools-extension v2.2.0.3.
